src/query/graphite/native/builtin_functions.go

```diff
@@ -242,6 +243,42 @@ func timeShift(
 	}, nil
 }
 
+
+func timeSlice(ctx *common.Context, inputPath singlePathSpec, start string, end string) (ts.SeriesList, error) {
+	tzOffsetForAbsoluteTime := time.Duration(0)
+	startTime, err := graphite.ParseTime(start, time.Now(), tzOffsetForAbsoluteTime)
```

Comment:
Instead of passing different `time.Now()` values, it's best to use a single time.Now() for both these time parsings so they see the same relative offset.

i.e. declare `now` first, then use them in both calls.